distribution/lib/Standard/Tableau/0.0.0-dev/src/Hyper_File.enso

```diff
@@ -0,0 +1,95 @@
+from Standard.Base import all
+import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
+from Standard.Base.Metadata.Choice import Option
+from Standard.Base.Metadata.Widget import Single_Choice
+
+from Standard.Table import Table
+import Standard.Table.Rows_To_Read.Rows_To_Read
+
+import project.Hyper_Table.Hyper_Table
+
+polyglot java import org.enso.tableau.HyperReader
+
+## Represents a Tableau Hyper Extract file.
+type Hyper_File
+    ## ICON data_input
+       Creates a Hyper_File
+
+       Arguments:
+       - file: The file to read.
+       - schema: The schema to read or `*` for all schemas.
+    new : File -> Text -> Hyper_File
+    new file:File schema:Text='*' = Hyper_File.Value file schema
+
+    ## PRIVATE
+       A representation of a Tableau Hyper Extract file.
+    private Value file:File internal_schema:Text
+
+    ## ICON metadata
+       Returns the list of schemas for the connection within the current database (or catalog).
+    schemas : Vector Text
+    schemas self =
+        array = HyperReader.readSchemas self.file.path
+        Vector.from_polyglot_array array
+
+    ## ICON metadata
+       Returns the name of the current schema.
+       `*` represents all schemas.
+    schema : Text
+    schema self = self.internal_schema
```

Comment:
What does it mean if `f.schema` returns `"*"`?

Reply:
It means accessing all tables - schema isn't used much inside Hyper but it is in theory supported.